cmd/lotus-shed/balances.go

```diff
@@ -125,6 +127,8 @@ var chainBalanceCmd = &cli.Command{
 				}
 				ai.Worker = info.Worker
 				ai.Owner = info.Owner
+				fmt.Printf("miner:%s, owner:%s, worker:%s, power:%s(%s)\n", ai.Address, ai.Owner,
```

Comment:
This command produces a CSV which is used in a number of places. This print breaks that output

Reply:
add miner-info flag, default:It won't print